events/jpa/src/main/java/org/keycloak/events/jpa/AdminEventEntity.java

```diff
@@ -0,0 +1,126 @@
+package org.keycloak.events.jpa;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+/**
+ * @author <a href="mailto:giriraj.sharma27@gmail.com">Giriraj Sharma</a>
+ */
+@Entity
+@Table(name="ADMIN_EVENT_ENTITY")
+public class AdminEventEntity {
+    
+    @Id
+    @Column(name="ID", length = 36)
+    private String id;
+    
+    @Column(name="ADMIN_EVENT_TIME")
+    private long time;
+    
+    @Column(name="OPERATION_TYPE")
+    private String operationType;
+    
+    @Column(name="REALM_ID")
+    private String realmId;
+    
+    @Column(name="CLIENT_ID")
+    private String clientId;
+
+    @Column(name="USER_ID")
```

Comment:
Can you prefix field from AuthDetails with 'auth' for example 'authUserId' so it's clear they are authentication details and not "resources being changed"